gem/styles/gem-springster/modules/_m-banner.scss

```diff
@@ -1,6 +1,10 @@
 
 .banner__item-link {
   display: block;
+  img {
```

Comment:
We should add a class for banner images here to reuse styles as much as possible. 
My bad